nucypher/config/storages.py

```diff
@@ -290,84 +290,6 @@ def initialize(self) -> bool:
         return not bool(self.__metadata or self.__certificates)
 
 
-class SQLiteForgetfulNodeStorage(ForgetfulNodeStorage):
```

Comment:
Any reason for this deletion? The NuCypher Status Monitor (`nucypher/nucypher-monitor`) uses this storage mechanism - https://github.com/nucypher/nucypher-monitor/blob/master/monitor/crawler.py

Reply:
What part of "Rip out as much SQLite testing stuff as I can" do you not understand?!??!

I'm kidding, but I was really ripping everything out that said SQLalchemy or SQLite as fast as I could haha. It can probably go back in, but perhaps we can replace a lot of it with the new datastore? Could it potentially get simplified?

Reply:
Perhaps, just be aware this is part of a different API around node storage, unrelated to the datastore.

Reply:
I'm not exactly sure I understand how it's being used. What is the reason we have a "forgetful" **SQLite** storage mechanism? Why not use a dict instead?

Reply:
It is forgetful because it is associated with data from the learning loop - which restarts from scratch every time the monitor is restarted.

In terms of why not use a `dict` - our queries are a little more involved than just (key, value) pairs and sqlite queries make that easier - not that it can't be done, but it is easier; remember as well that the Status Monitor is expected to run for a long time. For more insight into the queries used look in the `CrawlerStorageClient` class, see https://github.com/nucypher/nucypher-monitor/blob/master/monitor/db.py#L15.

Reply:
I think this class could be sensibly moved to the `nucypher-monitor` repo and maintained there. I don't see a reason to continue maintaining SQLite for complex queries we won't be performing in this repo.

Perhaps `nucypher-monitor` can grow to be a library for performing those more complex queries?

Reply:
We can leave SQLiteForgetfulNodeStorage for now. Filed #2118 as a follow-up.

Reply:
> "Rip out as much SQLite testing stuff as I can"
ditto except for this piece of old wallpaper ... 😅